tests/test_views.py

```diff
@@ -329,8 +329,8 @@ def _test_env_entry_render_params(bukudb, app, client, mode, favicons, new_tab,
     target = '' if not new_tab else ' target="_blank"'
     icon = '' if not favicons else (netloc and f'<img class="favicon" src="http://www.google.com/s2/favicons?domain={netloc}"/> ')
     urltext = title or '&lt;EMPTY TITLE&gt;'
-    _title = (urltext if not netloc else f'<a href="{url}"{target}>{urltext}</a>')
-    prefix = f'<td class="col-entry"> {icon}<span class="title">{_title}</span>'
+    _title = (urltext if not netloc and mode in ('full', None) else f'<a href="{url}"{target}>{urltext}</a>')
+    prefix = f'<td class="col-entry"> {icon}<span class="title" title="{url}">{_title}</span>'
```

Comment:
Since the test was added in a separate commit, the necessary change on implementation update is pretty transparent